packages/ember-application/tests/system/initializers_test.js

```diff
@@ -1,422 +1,396 @@
 import { run } from 'ember-metal';
-import Application from '../../system/application';
+import { assign } from 'ember-utils';
+import { moduleFor, AutobootApplicationTestCase, TestResolver } from 'internal-test-helpers';
+import { Application } from 'ember-application';
 import { jQuery } from 'ember-views';
 
-let app;
+moduleFor('Ember.Application initializers', class extends AutobootApplicationTestCase {
+  constructor() {
+    jQuery('#qunit-fixture').html(`
+      <div id="one">ONE</div>
+      <div id="two">TWO</div>
+    `);
+    super();
+  }
+
+  createApplication(options, MyApplication) {
```

Comment:
Using `applicationOptions` is preferred here. Replace `createApplication` through `teardown` with:

```js
 get applicationOptions() {
    return assign(super.applicationOptions, {
      rootElement: '#one'
    });
  }

  createSecondApplication(options, MyApplication=Application) {
    let myOptions = assign(this.applicationOptions, {
      rootElement: '#two'
    }, options);
    let secondApp = this.secondApp = MyApplication.create(myOptions);
    return secondApp;
  }

  teardown() {
    super.teardown();

    if (this.secondApp) {
      this.runTask(() => this.secondApp.destroy());
    }
  }
```

More details at https://github.com/emberjs/ember.js/pull/15362/files#r122541946